pkg/component/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -352,7 +352,8 @@ func (c *clusterAutoscaler) emptyManagedResource() *resourcesv1alpha1.ManagedRes
 }
 
 func (c *clusterAutoscaler) emptyManagedResourceSecret() *corev1.Secret {
-	return &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: managedresources.SecretName(managedResourceTargetName, true), Namespace: c.namespace}}
+	// TODO(dimityrmirchev): Remove this once mr secrets are turned into garbage-collectable, immutable secrets
```

Comment:
```suggestion
	// TODO(dimityrmirchev): Remove this once MR secrets are turned into garbage-collectable, immutable secrets
```
Can we add `v1.90` here too?